src/corpus/ui/ManuscriptPopover.tsx

```diff
@@ -60,7 +60,9 @@ export default function ManuscriptPopOver({
           {manuscript.period.name} {manuscript.period.description}
           <br />
         </p>
-        <ManuscriptReferences references={manuscript.references} />
+        <span className="manuscript-popover__references">
+          <ManuscriptReferences references={manuscript.references} />
+        </span>
```

Comment:
span should go inside ManuscriptReferences (unless its used in multiple places and only this one needs the span this ceratin way)